cloud/rackspace/rax_scaling_group.py

```diff
@@ -262,8 +262,8 @@ def rax_asg(module, cooldown=300, disk_config=None, files={}, flavor=None,
             # Launch Configuration Updates
             lc = sg.get_launch_config()
             lc_args = {}
-            if server_name != lc.get('name'):
-                lc_args['name'] = server_name
+            if server_name != lc.get('server_name'):
```

Comment:
While `update_launch_config` does indeed specify `server_name` as the keyword argument, the data from `get_launch_config` uses `name`.  So line 265 should stay the same, only 266 should change.

`get_launch_config` return data:

```
{
    'name': 'ansible-testing-Lqge0qB6-1',
    'image': '753a7703-4960-488b-aab4-a3cdd4b276dc',
    'networks': [
        {'uuid': '00000000-0000-0000-0000-000000000000'},
        {'uuid': '11111111-1111-1111-1111-111111111111'}
    ],
    'load_balancers': [],
    'personality': [],
    'type': 'launch_server',
    'flavor': 'performance1-1',
    'metadata': {}
}
```